client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -144,46 +158,57 @@ class EditorConfirmationSidebar extends Component {
 		const isSidebarActive = this.props.status === 'open';
 		const isOverlayActive = this.props.status !== 'closed';
 
-		return (
-			<div
-				className={ classnames( {
-					'editor-confirmation-sidebar': true,
-					'is-active': isOverlayActive,
-				} ) }
-			>
-				{ this.renderPublishingBusyButton() }
+		if ( ! this.state.hasUnclosed ) {
+			return (
+				<div className="editor-confirmation-sidebar">
+					<div key="sidebar" className="editor-confirmation-sidebar__sidebar" />
+				</div>
+			);
+		}
 
+		return (
+			this.state.hasUnclosed && (
```

Comment:
On this line, it's already guaranteed that `hasUnclosed` is true, isn't it?

Reply:
Yup, good point!